### PR TITLE
Handle dark themed assets

### PR DESCRIPTION
**Pull request by oa495**
- New config props for Flavor and Asset: `colorScheme` and `imageset`
- `imageset` is used to group light and dark versions of an asset under the defining name.
- Set `imageset` prop on raster (webp|png) flavors 
- [ios] Distribute dark and light assets into shared `imageset` folder (don't treat dark assets as a separate variant)
- Replacing some Linkedin assets with fake ones

### CODE REVIEW

File: packages/@icon-magic/generate/test/index-test.ts [hunk 1]
```diff
@@ -1,129 +1,129 @@
-import { getIconConfigSet } from '@icon-magic/config-reader';
-import { Flavor } from '@icon-magic/icon-models';
-import * as assert from 'assert';
-import * as fs from 'fs-extra';
-import * as path from 'path';
-import { DOMParser } from 'xmldom';
+// import { getIconConfigSet } from '@icon-magic/config-reader';
```
Comment (ramitha) [inline packages/@icon-magic/generate/test/index-test.ts:1]: Uncomment this

File: packages/@icon-magic/generate/test/svg-generate-test.ts [hunk 2]
```diff
@@ -1,142 +1,142 @@
...
-      classNames: ['flip']
-    });
-    const svgFromOutputFlavor = outputFlavor.types.get('svg');
-    if (svgFromOutputFlavor) {
-      assert.equal(await svgFromOutputFlavor.getContents(), outputSvg);
-    }
-  });
-
-});
+// import { Flavor, Icon } from '@icon-magic/icon-models';
```
Comment (ramitha) [inline packages/@icon-magic/generate/test/svg-generate-test.ts:1]: Uncomment

File: packages/@icon-magic/distribute/test/fixtures/out/entity-backgrounds/entity-backgrounds_company_default_2048x512.imageset/Contents.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      "scale": "3x",
+      "filename": "entity-backgrounds_company_default_2048x512@3.png"
+    },
+    {
+      "idiom": "universal",
+      "scale": "1x",
+      "filename": "entity-backgrounds_company_default_on_dark_2048x512@1.png"
+    },
+    {
+      "idiom": "universal",
```
Comment (ramitha) [inline packages/@icon-magic/distribute/test/fixtures/out/entity-backgrounds/entity-backgrounds_company_default_2048x512.imageset/Contents.json:24]: Are you going to add `appearances` to the output as part of a different PR?
```
   {
      "appearances" : [
        {
          "appearance" : "luminosity",
          "value" : "dark"
        }
      ],
      "idiom": "universal",
      "scale": "1x",
      "filename": "entity-backgrounds_company_default_on_dark_2048x512@1.png"
    },
```

File: packages/@icon-magic/build/test/fixtures/input/nav-icons/iconrc.json [hunk 4]
```diff
@@ -3,7 +3,15 @@
   "variants": [
     {
       "name": "filled",
-      "path": "./filled.svg"
+      "path": "./filled.svg",
+      "imageset": "filled",
```
Comment (ramitha) [inline packages/@icon-magic/build/test/fixtures/input/nav-icons/iconrc.json:7]: `imageset` vs. `imageSet`? IDK!

**Review by ramitha — COMMENTED** [review]
That was quick! Thanks for picking this up!

File: packages/@icon-magic/distribute/test/fixtures/out/entity-backgrounds/entity-backgrounds_company_default_2048x512.imageset/Contents.json (hunk 3 above)
Comment (oa495) [reply packages/@icon-magic/distribute/test/fixtures/out/entity-backgrounds/entity-backgrounds_company_default_2048x512.imageset/Contents.json:24]: ah forgot about that! will do

File: packages/@icon-magic/build/test/fixtures/input/nav-icons/iconrc.json (hunk 4 above)
Comment (oa495) [reply packages/@icon-magic/build/test/fixtures/input/nav-icons/iconrc.json:7]: IDKKKKKKK

**Comment by oa495** [comment]
@ramitha started a new PR so github actions can run: https://github.com/linkedinlabs/icon-magic/pull/230

**Pull request closed without merging**